adafruit_esp32spi/adafruit_esp32spi_wifimanager.py

```diff
@@ -211,3 +211,11 @@ def pixel_status(self, value):
         """
         if self.statuspix:
             self.statuspix.fill(value)
+
+    def signal_strength(self):
+        """
+        Returns receiving signal strength indicator in dBm
+        """
+        if not self._esp.is_connected:
+            self.connect()
+        return self._esp.rssi()
```

Comment:
In discord you mentioned that `_esp.rssi()` returns an unpacked `struct` and would need to be cleaned up; is that not the case?